libraries/helpers.rb

```diff
@@ -0,0 +1,22 @@
+module RabbitMQCookbook
+  module Helpers
+    require 'mixlib/shellout'
+
+    def remove_self_from_cluster
```

Comment:
This name is not particularly correct. You are resetting the node here, which does not notify the rest of the cluster that the node is gone (and can be used in more scenarios).